.gitignore

```diff
@@ -1,3 +1,4 @@
 android/build
 android/local.properties
 android/.gradle
+android/app/app.iml
```

Comment:
Missing final newline. You may want to change settings in IDE or git to prevent further problems.

Reply:
fixed